lang/syn/src/codegen/accounts/constraints.rs

```diff
@@ -630,31 +656,52 @@ pub fn generate_init(
     }
 }
 
+pub fn generate_create_account(
+    field: &Ident,
+    space: proc_macro2::TokenStream,
+    owner: proc_macro2::TokenStream,
+    seeds_with_nonce: proc_macro2::TokenStream,
+) -> proc_macro2::TokenStream {
+    return generate_create_account_lamports(
+        field,
+        space,
+        proc_macro2::TokenStream::new(),
+        owner,
+        seeds_with_nonce,
+    );
+}
+
 // Generated code to create an account with with system program with the
 // given `space` amount of data, owned by `owner`.
 //
 // `seeds_with_nonce` should be given for creating PDAs. Otherwise it's an
 // empty stream.
-pub fn generate_create_account(
+pub fn generate_create_account_lamports(
     field: &Ident,
     space: proc_macro2::TokenStream,
+    lamports: proc_macro2::TokenStream,
```

Comment:
This should probably be an Option considering the use of `proc_macro2::TokenStream::new()` above.